src/test/java/org/symphonyoss/symphony/messageml/elements/form/FormTest.java

```diff
@@ -54,7 +54,16 @@ public void testFormWithInvalidChild() throws Exception {
     expectedException.expectMessage("Invalid MessageML content at element \"invalid-child\"");
 
     context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
-    
+  }
+
+  @Test
```

Comment:
Please include another test with a form element inside a div inside another form element so we know the parentValidation is working as expected for other elements within the tree.

Reply:
@cristiadu the `testNestedForms()` method below does it. :)

Reply:
Weird, it was showing the old code to me for some reason.